index.bs

```diff
@@ -416,6 +419,211 @@ The <dfn method for=FileSystemHandle>isSameEntry(|other|)</dfn> method steps are
+       |destinationDirectory|'s [=FileSystemHandle/locator=].
+    1. Let |destinationDirectoryEntry| be the result of [=locating an entry=]
+       given |destinationDirectoryLocator|.
+
+    1. Let |destinationDirectoryAccessResult| be the result of running
+       |destinationDirectoryEntry|'s
+       [=file system entry/request access=] given "`readwrite`".
+    1. If |destinationDirectoryAccessResult|'s
+       [=file system access result/permission state=]
+       is not "{{PermissionState/granted}}", [=queue a storage task=] with
+       |global| to [=/reject=] |result| with a {{DOMException}} of
+       |accessResult|'s [=file system access result/error name=] and
+       abort these steps.
+
+    1. If |destinationDirectoryLocator|'s [=file system locator/root=] is not
+       |locator|'s [=file system locator/root=], [=queue a storage task=] with
+       |global| to [=/reject=] |result| with an
+       "{{NotSupportedError}}" {{DOMException}} and abort these steps.
+
+       Issue(114): Decide which moves across file systems, if any, should be
+       supported, or if this should be left up to individual user-agent
+       implementations.
+
+    1. Let |destinationPath| be the result of [=list/clone|cloning=]
+       |destinationDirectoryLocator|'s [=file system locator/path=] and
+       [=list/append|appending=] |newEntryName|.
+  1. Otherwise:
+    1. Let |destinationPath| be the result of [=list/clone|cloning=]
+       |locator|'s [=file system locator/path=], [=list/remove|removing=]
+       the last [=list/item=], and [=list/append|appending=] |newEntryName|.
+
+  1. Let |destinationLocator| be a [=/file system locator=] whose
+     [=file system locator/kind=] is |locator|'s [=file system locator/kind=],
+     [=file system locator/root=] is |locator|'s [=file system locator/root=], and
+     [=file system locator/path=] is |destinationPath|.
+  1. Let |destinationEntry| be the result of [=locating an entry=]
+     given |destinationLocator|.
+
+  1. If |destinationDirectory| was not given:
+    1. Let |destinationAccessResult| be the result of running
+       |destinationEntry|'s [=file system entry/request access=]
+       given "`readwrite`".
+
+       Issue(101): Make sure this still makes sense once access check algorithms
+       are no longer associated with an entry.
+  1. Otherwise:
+    1. Let |destinationAccessResult| be |destinationDirectoryAccessResult|.
+
+  1. Let |isTheRootOfABucketFileSystem| be true if
+     |isInABucketFileSystem| is true and
+     |entry|'s [=file system entry/parent=] is `null`;
+     otherwise false.
+  1. If |isTheRootOfABucketFileSystem| is true,
+     [=queue a storage task=] with |global| to [=/reject=] |result| with a
+     "{{InvalidModificationError}}" {{DOMException}} and abort these steps.
+
+  Issue: Do not allow |entry| to be moved if it is a
+  <a href=https://wicg.github.io/file-system-access/#wellknowndirectory-too-sensitive-or-dangerous>sensitive directory</a>
```

Comment:
Whoops, I meant to say that [well-known directories](https://wicg.github.io/file-system-access/#enumdef-wellknowndirectory) should not be moved. The site should never obtain a handle to a sensitive directory in the first place